src/controllers/todo.controller.ts

```diff
@@ -4,10 +4,13 @@ import {repository} from '@loopback/repository';
 import {TodoRepository} from '../repositories/index';
 
 export class TodoController {
-  constructor(@repository('todo') protected todoRepo: TodoRepository) {}
+  // TODO(bajtos) Fix documentation (and argument names?) of @repository()
+  // to allow the usage below.
+  constructor(@repository(TodoRepository.name) protected todoRepo: TodoRepository) {}
```

Comment:
`app.repository(TodoRepository)` binds the repository constructor as `repositories.TodoRepository`, thus this line could have been changed to `@repository('TodoRepository')`. However, I'd like to avoid string constants because don't follow "refactor - rename" changes. `TodoRepository.name` returns the string name of the repository we want to inject. If we decide to rename the repository class to something else, then refactoring tools will update `TodoRepository.name` to `MyNewRepository.name`.

The only confusing part is that `@repository` says the first string argument should be a model name. I think `@repository` should allow repository name too and we should change the API and apidocs to support that (see the comment left in this file).